helper.py

```diff
@@ -157,3 +157,14 @@ def preds_prob(df, preds, true, dataset, output_path):
     plt.show()
     fig.tight_layout()
     fig.savefig(str(output_path)+'/test_best_pred.png')
+
+
+def diff_SB(df, signal_label):
+    dfs = df[df[signal_label]==1]
+    dfb = df[df[signal_label]==0]
```

Comment:
I would try something like 
`
dfb = df[df[signal_label]==0]
if len(df[signal_label].unique()) < 3:
      dfs = df[df[signal_label]==1]
else:
    dfsp = df[df[signal_label]==1]
    dfss = df[df[signal_label]>1]
 `

Reply:
Could you please explain more explicitly these changes?